TheaterSchedule.BLL/Services/GetDataFromGoogleFormService.cs

```diff
@@ -0,0 +1,91 @@
+﻿using HtmlAgilityPack;
+using TheaterSchedule.BLL.Classes;
+using System.Net;
+using TheaterSchedule.BLL.Interfaces;
+namespace TheaterSchedule.BLL.Services
+{
+    public class GetDataFromGoogleFormService:IGetDataFromGoogleFormService
+    {
+        private string rootUrl;
+
+        public static string GetUrl(string address)
+        {
+            WebClient webClient = new WebClient();
+            webClient.Headers[HttpRequestHeader.Accept] = "text/html, */*";
+            webClient.Headers[HttpRequestHeader.AcceptLanguage] = "ru-RU";
+            webClient.Headers[HttpRequestHeader.UserAgent] =
+                "Mozilla/5.0 (compatible; MSIE 9.0; Windows NT 6.1; Trident/5.0)";
+            webClient.Headers[HttpRequestHeader.ContentType] = "application/x-www-form-urlencoded,  charset=utf-8";
+            return webClient.DownloadString(address);
+        }
+
+        public Form GetDataFromServer(string rootUrl)
+        {
+            this.rootUrl = rootUrl.Replace("viewform", "formResponse?");
+
+            HtmlDocument htmlSnippet = new HtmlDocument();
+            string result;
+
+            result = GetUrl(rootUrl);
+            htmlSnippet.LoadHtml(result);
+            HtmlNodeCollection nodes =
+                htmlSnippet.DocumentNode.SelectNodes(
+                    @"//div[@class='freebirdFormviewerViewNumberedItemContainer']");
+
+            Form forms = new Form();
+            foreach (HtmlNode node in nodes)
+            {
+                string title = node
+                    .SelectSingleNode(
```

Comment:
Google has api for work with forms

https://developers.google.com/apps-script/reference/forms/
Parsing of HTML is not reliable (it will be broken when some html layout will be changed)